assets/js/components/ActivateModuleCTA.js

```diff
@@ -62,7 +62,7 @@ const ActivateModuleCTA = ( { moduleSlug, title, description } ) => {
 				type: 'win-error',
 			} );
 		}
-	}, [ activateModule ] );
+	}, [ activateModule, navigateTo, moduleSlug ] );
```

Comment:
most of this PR are `useCallback` dependency arrays being populated with all the dependencies that eslint tells us